apps/protocol-frontend-e2e/src/integration/02_waitlist.cy.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+
+const network = "goerli";
+const address: string = Cypress.env('address');
+const COOKIE: string = Cypress.env('COOKIE');
+
+const truncateTablesQuery = `
+TRUNCATE TABLE "GuildContribution" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "Contribution" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "User" RESTART IDENTITY CASCADE;
+TRUNCATE TABLE "Guild" RESTART IDENTITY CASCADE;
+`
```

Comment:
Let's not trucate the table our tests should create data and tear it down rather than removing all the data in the table. Even if we don't teardown. It would be good to append a hash on unique strings so we aren't using the same variables.

Reply:
I don't think I understand this comment.  
The intention of the queries above was to clear the tables mentioned so we can assume in our tests that we have a new user, who will also be creating their first Contribution. For us to test for a new user and a first contribution, don't we need to have the DB cleared initially?